### ISSUE TITLE
About NTU60 and NTU120 dataset

### ISSUE THREAD

**Issue by HaleYan** [kennymckormick/pyskl#90]
Thanks for your great work !
In the experiment, you have done the experiment of NTU60-XSub, NTU60-XView and NTU120-XSub, NTU120-XSet and presented the results in the paper, but only NTU60-XSub and NTU120-XSub are provided [mmaction2/tools/data/skeleton/README](https://github.com/open-mmlab/mmaction2/tree/master/tools/data/skeleton), so I want to know How the results of NTU60-XView and NTU120-XSet are obtained. And in [mmaction2/tools/data/skeleton/README](https://github.com/open-mmlab/mmaction2/tree/master/tools/data/skeleton), how did you divide the training set and test set of the hmdb51 and ucf101 datasets. thanks

**Comment by HaleYan** [comment]
@kennymckormick Hi,Thanks for your reply, but now I want to get ntu60_xview_train_pkl,ntu60_xview_val_pkl,ntu120_xset_train.pkl,ntu120_xset_val.pkl, can you provide it?

**Comment by kennymckormick** [comment]
Please refer to this documentation: https://github.com/kennymckormick/pyskl/blob/main/tools/data/data_doc.md. For example, for ntu60_xview, you can directly download the ntu60_3danno.pkl. This pickle file includes all NTU60 annotations and the split information for both NTU60-XSub and NTU60-XView. Once you download it, put it under the 'data/nturgbd' directory, you can train and test NTU60-XView. Same for NTU120-XSet.
